### PR TITLE
Prebid 7: TheMediaGrid: make trustxBidAdapter as alias of gridBidAdapter

### PR DESCRIPTION
**Pull request by TheMediaGrid**
## Type of change

- [x] Feature
- [x] Refactoring (no functional changes, no api changes)

## Description of change

Make trustxBidAdapter as alias of gridBidAdapter

- grid-tech@themediagrid.com
- [x] official adapter submission

- A link to a PR on the docs repo https://github.com/prebid/prebid.github.io/pull/3727

### CODE REVIEW

**Comment by patmmccann** [comment]
tagging #7796 for linkage

File: modules/gridBidAdapter.js [hunk 1]
```diff
@@ -136,8 +151,19 @@ export const spec = {
       if (impObj.banner || impObj.video) {
         imp.push(impObj);
       }
+
+      if (!forceBidderName && forceBidder && ALIAS_CONFIG[forceBidder]) {
+        forceBidderName = forceBidder;
+      }
     });
 
+    forceBidderName = config.getConfig('forceBidderName') || forceBidderName;
```
Comment (patmmccann) [inline modules/gridBidAdapter.js:160]: please follow the conventions here

https://github.com/prebid/Prebid.js/issues/8129

**Review by patmmccann — CHANGES_REQUESTED** [review]

**Comment by patmmccann** [comment]
I changed the base to prebid 7 bc this deletes trustx

**Comment by patmmccann** [comment]
@TheMediaGrid any thoughts on https://github.com/prebid/Prebid.js/pull/8321#discussion_r861996325

File: modules/gridBidAdapter.js (hunk 1 above)
Comment (TheMediaGrid) [reply modules/gridBidAdapter.js:160]: Hello @patmmccann ! In this PR TrustX adapter migrate to alias for grid bid adapter and when gird bid adapter catch bidderCode === 'trustx', it sent request with alternative parameters. Can you suggest any solution to  to fix case, when publisher set alias for trustx - pbjs.aliasBidder('trustx', 'mybidder'); and we need to sent request as for trustx? How grid bid adapter can find that 'mybidder' is alias exactly for 'trustx', not for 'grid'?

Comment (TheMediaGrid) [reply modules/gridBidAdapter.js:160]: If i right understand this feature https://github.com/prebid/Prebid.js/issues/8129 can't resolve this problem.

**Comment by TheMediaGrid** [comment]
> @TheMediaGrid any thoughts on [#8321 (comment)](https://github.com/prebid/Prebid.js/pull/8321#discussion_r861996325)

Hello @patmmccann ! Please view comment https://github.com/prebid/Prebid.js/pull/8321#discussion_r872566356

**Comment by patmmccann** [comment]
@TheMediaGrid can you please resolve the conflicts

**Review by patmmccann — APPROVED** [review]

File: modules/gridBidAdapter.js (hunk 1 above)
Comment (patmmccann) [reply modules/gridBidAdapter.js:160]: Tyvm

**Pull request merged**